controllers/api.controller.js

```diff
@@ -45,4 +45,22 @@ exports.getCommentsByArticleId = (req, res, next) => {
 };
 
 
+exports.postCommentToArticle = (req, res, next) => {
+  
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+ 
+
+  if (!username || !body) {
+    return res.status(400).send({ msg: "not found" });
+  }
```

Comment:
This does work, but ideally error handling should be handled by the model instead of here